insteon_mqtt/device/base/Base.py

```diff
@@ -1149,6 +1175,16 @@ def handle_flags(self, msg, on_done):
                "{:08b}".format(msg.cmd2))
         on_done(True, "Operation complete", msg.cmd2)
 
+    #-----------------------------------------------------------------------
+    def handle_std_raw_message(self, msg, on_done):
```

Comment:
I'll probably just combine these handlers, I want sure if the response/signature was different based on standard/extended messages